server/notification-providers/teams.js

```diff
@@ -57,41 +60,114 @@ class Teams extends NotificationProvider {
 
         const facts = [];
 
+        if (monitorMessage) {
+            facts.push({
+                title: "Description",
+                value: monitorMessage,
+            });
+        }
+
         if (monitorName) {
             facts.push({
-                name: "Monitor",
+                title: "Monitor",
                 value: monitorName,
             });
         }
 
         if (monitorUrl && monitorUrl !== "https://") {
             facts.push({
-                name: "URL",
-                value: monitorUrl,
+                title: "URL",
+                // format URL as markdown syntax, to be clickable
+                value: `[${monitorUrl}](${monitorUrl})`,
             });
         }
 
-        return {
-            "@context": "https://schema.org/extensions",
-            "@type": "MessageCard",
-            themeColor: this._getThemeColor(status),
-            summary: notificationMessage,
-            sections: [
-                {
-                    activityImage:
-                        "https://raw.githubusercontent.com/louislam/uptime-kuma/master/public/icon.png",
-                    activityTitle: "**Uptime Kuma**",
-                },
-                {
-                    activityTitle: notificationMessage,
-                },
+        const headerMessage = `**${notificationMessage}**`;
+
```

Comment:
No need to extract this imo

```suggestion
```

Reply:
I had some issue with the inline string evaluation. But it might've been due to another issue. I've inlined it now.